api/restapi.py

```diff
@@ -368,8 +368,8 @@ def add_social_account(self, request):
         try:
             p = Profile(self.db)
             if "account_type" in request.args and "username" in request.args and "proof" in request.args:
-                p.add_social_account(request.args["account_type"][0], request.args["username"][0],
-                                     request.args["proof"][0] if "proof" in request.args else None)
+                p.add_social_account(request.args["account_type"][0], request.args["username"][0].decode('utf8'),
+                                     request.args["proof"][0].decode('utf8') if "proof" in request.args else None)
```

Comment:
Updated. Please note that there are:
* 19 E402: module level import not at top of file
* 105 E501 line too long

in that single file